scripts/l4t-version.sh

```diff
@@ -0,0 +1,49 @@
+#!/bin/bash
```

Comment:
I don't see this script used.  Can we remove?

Reply:
This is tied to the change above to build.sh.
If we don't take that change above, we don't need this.